crates/noirc_frontend/src/ast/structure.rs

```diff
@@ -1,6 +1,7 @@
 use std::fmt::Display;
 
-use crate::{Ident, NoirFunction, Path, UnresolvedType};
+use crate::{Ident, NoirFunction, UnresolvedGenerics, UnresolvedType};
+use iter_extended::vecmap;
 use noirc_errors::Span;
 
 #[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
Can you add doc comments to this and its public fields? Not a blocker since this struct was not created in this PR

Reply:
I've added a short comment explaining it is an Ast node for structs but do not think there is much else to say